sources/INSTALL.md

```diff
@@ -25,17 +30,29 @@ depending on whether ocamlopt is installed or only ocamlc is detected.
 
   3. Uninstall with "make uninstall"
```

Comment:
I don't remember if make uninstall works if we only install some targets ? To be tested

Reply:
Ah ! I see below that you also added targets to remove specific packages